HomeAssistant/AppDelegate.swift

```diff
@@ -892,24 +892,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func setupFirebase() {
-        var fileName = "GoogleService-Info-Debug"
+        LogDestination = CrashlyticsLogDestination()
```

Comment:
could do…

```suggestion
        Current.Log.add(CrashlyticsLogDestination())
```

Reply:
I don’t think that’ll work since `Log` is an initialized variable...